dev-docs/RFCs/v1.0/tentative-feature.md

```diff
@@ -0,0 +1,59 @@
+# Tentative Features
+
+In order to better support Multi* geometries, I think we'll need a concept like "tentative feature". For example, when the user adds two points of what will be a second polygon of a MultiPolygon feature. This concept would only apply to the draw* modes.
+
+The term "tentative" makes it clear that it isn't a real thing yet. Tentative features will need to stick around between `onEdit` calls, so it can't just be the `updatedData` argument. It'll need its own state/prop.
+
+Here's my thought:
+
+```
+new EditableGeoJsonLayer({
+  data: this.state.data,
+  onEdit: ({data}) => this.setState({data}), // just as onEdit functions today
+
+  // optional: if caller wants to customize tentative feature handling
```

Comment:
This customization, would be good to have some code examples, what will it be used for?

Does it allow the user to render his own tentative feature, or just modify the props, etc?